nautobot/extras/tests/test_migrations.py

```diff
@@ -154,3 +159,76 @@ def test_custom_field_data_populated_correctly(self):
                 "text_custom_field_4": None,
             },
         )
+
+
+@tag("migration_test")
+class TestFailingApprovalMigration(TestCase):
```

Comment:
Shouldn't this inherit from `NautobotDataMigrationTest`? Or is this done differently because the migration is expected to abort?